lms/migrations/versions/a9e5a33e96b7_backfill_lmsuserassignmentmembership.py

```diff
@@ -0,0 +1,50 @@
+"""Backfill LMSUserAssignmentMembership."""
+
+import sqlalchemy as sa
+from alembic import op
+
+revision = "a9e5a33e96b7"
+down_revision = "712c4c9a4e2e"
+
+
+def upgrade() -> None:
+    conn = op.get_bind()
+    conn.execute(
+        sa.text(
+            """
+        WITH backfill as (
+            SELECT
```

Comment:
Here backfill is a relatively simple query so the CTE (WITH backfill as...) adds a bit repetition but I recon is easier to ready anyway.